src/app/elements/ParametersViewElement.js

```diff
@@ -156,6 +177,59 @@ export default class ParametersViewElement extends LitElement {
     }));
   }
 
+  [$onSave] = (e) => {
+    e.preventDefault();
+    savedData = (this.entities && this.entities[this.uuid]) || null;
+    copyInfo = clone(savedData);
+    console.log('Transform data has been saved!');
+  };
+
+  [$displayData] = (e, subProps, elements) => {
+    e.preventDefault();
+    const copy2 = clone(copyInfo);
```

Comment:
Are you cloning twice (once in onSave and once in displayData) because copyInfo acts as a cache for saved data and copy2 only contains the data that'll be rendered? I was a little confused why you cloned twice.